src/helpers.js

```diff
@@ -77,19 +77,20 @@ async function getAppInfo() {
 
   const assets = await appInstance.assets.list();
 
-  const passcodeVar = variables.find(v => v.key === 'API_PASSCODE');
-  const expiryVar = variables.find(v => v.key === 'API_PASSCODE_EXPIRY');
+  // const passcodeVar = variables.find(v => v.key === 'API_PASSCODE');
```

Comment:
I kept a bunch of this stuff commented out and didn't delete it until I've been able to do enough rounds of production testing to be sure I didn't make the entire world collapse.